tst/testextra/ctblpope.tst

```diff
@@ -0,0 +1,61 @@
+#############################################################################
+##
+#W  ctblpope.tst                GAP tests                       Thomas Breuer
+##
+#Y  Copyright (C) 2017
+##
+##  excluded from 'testinstall.g' as it takes considerable time
+##
+
+gap> START_TEST( "ctblpope.tst" );
+
+##
+gap> m:= CharacterTable( "M" );;
```

Comment:
Before calling `CharacterTable( "M" );;`, please put the line
```
gap> LoadPackage("ctbllib", false);;
```
This will ensure that the Character Table Library is loaded for the test, otherwise `make testandard` will fail when no packages are loaded with this being the first of a number of diffs: 
```
########> Diff in /circa/scratch/gap-jenkins/workspace/GAP-pull-request-quickt\
est/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/standard/label/kovacs/GAP-pull-reque\
st-snapshot/tst/testextra/ctblpope.tst:13
# Input is:
m:= CharacterTable( "M" );;
# Expected output:
# But found:
Error, sorry, the GAP Character Table Library is not loaded,
call `LoadPackage( "CTblLib" )' if you want to use it
########
```

Reply:
P.S. A better idea would be indeed to run the test only when `IsPackageMarkedForLoading("ctbllib","");` returns `true`. This will not enforce loading the package when GAP is started with `-r -A` options.

Reply:
e.g. along the lines
```
gap> if IsPackageMarkedForLoading("ctbllib","") then
... do calculations
... if Length(cand)<>1 then
>   Print( "Error in calculation of ...\n" );
> fi; fi;
```